ssv/aggregator.go

```diff
@@ -50,16 +50,16 @@ func NewAggregatorRunner(
 	}
 }
 
-func (r *AggregatorRunner) StartNewDuty(duty *types.Duty) error {
-	return r.BaseRunner.baseStartNewDuty(r, duty)
+func (r *AggregatorRunner) StartNewDuty(duty types.Duty, quorum uint64) error {
```

Comment:
Better name would be "threshold" here instead of quorum (since it just affects signatures)

When separating `Share`  to `Share` (used for partial sigs) and `CommitteeMember` (used for consensus), I noticed we don't use `Share.Quorum` at all for post consensus. What we use is the `PartialSigContainer.Quorum`.
So I took quorum out of `Share` since it was only used to pass down the value to `PartialSigContainer`.
Now I had to init PartialSigContainer another way.